ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -128,16 +121,24 @@ then
 	then
 		if fmu mode_pwm
 		then
-			if mixer load $OUTPUT_AUX_DEV $MIXER_AUX_FILE
+			if [ -f $OUTPUT_AUX_DEV ]
```

Comment:
If you use the -e operator instead of -f it works (-f checks for regular files only).